app/main/views/brief_responses.py

```diff
@@ -82,6 +84,54 @@ def create_brief_response():
     return jsonify(briefResponses=brief_response.serialize()), 201
 
 
+@main.route('/brief-responses/<int:brief_response_id>/submit', methods=['POST'])
+def submit_brief_response(brief_response_id):
+    json_payload = get_json_from_request()
+    updater_json = validate_and_return_updater_request()
+
+    brief_response = BriefResponse.query.filter(
+        BriefResponse.id == brief_response_id
+    ).first_or_404()
+
+    brief = brief_response.brief
+    supplier = brief_response.supplier
+    brief_service = get_supplier_service_eligible_for_brief(supplier, brief)
+
+    if brief_response.status != 'draft':
+        abort(400, "Brief response must be a draft")
+
+    if brief.framework.status != 'live':
+        abort(400, "Brief framework must be live")
+
+    brief_role = brief.data["specialistRole"] if brief.lot.slug == "digital-specialists" else None
```

Comment:
I'm really wary of these hard-coded DOS-specific lot names and question keys.  The API shouldn't need to know about these things.

It doesn't really seem scalable to brief building being able to work for "any framework that needs buyers to specify requirements", which is supposed to be the goal.  (I know these are already in the `create_brief_response` route too, but I think it's worth mentioning again that at some point we're going to need to find a way to avoid doing this, perhaps somehow passing information from the frontend app that tell the API what general-purpose rules to apply).